todo-app/src/App.js

```diff
@@ -1,13 +1,63 @@
 import React, { Component } from 'react';
+import Task from './Task';
+import TaskForm from './TaskForm';
+import tasksStore from './TasksStore';
+import {observer} from 'mobx-react';
+
+@observer
+export default class App extends Component {
+
+  constructor(props) {
+		super(props);
+	
+    this.state = {
```

Comment:
Sorry, I guess we missed this last week... did you have a reason for leaving this state inside a React component instead of putting it inside your MobX store? I think it would be better to move it inside the store.